src/ResourceManagement/Compute/Generated/Models/DataDisk.cs

```diff
@@ -174,7 +174,7 @@ public DataDisk(int lun, DiskCreateOptionTypes createOption, string name = defau
         /// Gets or sets the managed disk parameters.
         /// </summary>
         [JsonProperty(PropertyName = "managedDisk")]
-        public Management.ResourceManager.Fluent.SubResource ManagedDisk { get; set; }
+        public ManagedDiskParametersInner ManagedDisk { get; set; }
```

Comment:
What causes this change? What's in swagger?

Reply:
I think it's generator issue, already existing when we did upgrade last time [here](https://github.com/Azure/azure-libraries-for-net/pull/911/commits/bc14e74928c0e67c97b474da4cfafc2d87aafaf3)